src/3d/symbols/qgspoint3dsymbol.cpp

```diff
@@ -100,6 +114,21 @@ QString QgsPoint3DSymbol::shapeToString( QgsPoint3DSymbol::Shape shape )
     case Plane: return QStringLiteral( "plane" );
     case ExtrudedText: return QStringLiteral( "extruded-text" );
     case Model: return QStringLiteral( "model" );
+    case Billboard: return QStringLiteral( "billboard" );
     default: Q_ASSERT( false ); return QString();
   }
 }
+
+QMatrix4x4 QgsPoint3DSymbol::billboardTransform() const
+{
+
+  float *md = transform().data();
+
+  QMatrix4x4 billboardTransformMatrix;
+  billboardTransformMatrix.translate( QVector3D( 0, md[13], 0 ) );
+  billboardTransformMatrix.scale( QVector3D( 1, 1, 1 ) );
+  billboardTransformMatrix.rotate( QQuaternion::fromEulerAngles( 0, 0, 0 ) );
```

Comment:
there should be no need to set unit scale and no rotation to the matrix - that should be the default already.